### ISSUE TITLE
[Tab tray UI experiment] [Accessibility] -  The tabs titles are not read as buttons by voice over

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#25828]
## Steps to reproduce
1. Enable the Voice Over from iOS settings.
1. Open Firefox 
1. Access tab tray.
1. Observe the section behaviour.

#### **Actual Result:**
The tab titles are not announced with voice over as buttons.

#### **Expected Result:**
The tab titles are announced as buttons.

### Device & build information
* Device: iPhone 14 Pro Max (iOS 16.2).
* Build version: v9000 (53263)

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/FXIOS-11856)

**Comment by data-sync-user** [comment]
➤ Adina Petridean commented:

Verified as fixed on v9000( 53622) with iPhone 14 Pro Max (iOS 16.2).

[^RPReplay_Final1744357994.MP4]

**Comment by data-sync-user** [comment]
➤ Adina Petridean commented:

Verified as fixed on Firefox 139( 54805) with iPhone 14 Pro Max (iOS 16.2).
